nemo/lightning/pytorch/plugins/precision.py

```diff
@@ -156,4 +252,27 @@ def forward_context(self) -> Generator[None, None, None]:
             pass
 
 
-__all__ = ["MegatronMixedPrecision"]
+class MegatronMixedPrecision(MegatronCustomPrecision):
+    def __init__(
+        self,
+        precision: Literal["16-mixed", "bf16-mixed"],
+        amp_O2: bool = False,
```

Comment:
amp_02 can be removed since we don't support it anymore

Reply:
thanks I'll have that removed once https://github.com/NVIDIA/NeMo/pull/10080 is merged.